components/analytics/OETHPriceChart.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { last } from "lodash";
+import {
+  CategoryScale,
+  Chart as ChartJS,
+  LinearScale,
+  LineElement,
+  PointElement,
+  TimeScale,
+  Title,
+  Tooltip,
+  Legend,
+} from "chart.js";
+import { Line } from "react-chartjs-2";
+import { useOETHPriceChart } from "../../hooks/analytics/useOETHPriceChart";
+import { formatCurrency } from "../../utils";
+import { chartOptions } from "../../utils/analytics";
+import LayoutBox from "../LayoutBox";
+import DefaultChartHeader from "./DefaultChartHeader";
+import DurationFilter from "./DurationFilter";
+
+ChartJS.register(
```

Comment:
I think this is normally correct but I have noticed issues with bundling that it doesn't work and errors sometimes depending on how someone enters the app. That is unless in `_app` we just add all the ChartJs registrations. This can be done but isn't how its setup right now